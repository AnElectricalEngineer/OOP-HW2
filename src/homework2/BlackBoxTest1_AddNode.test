# Black Box Test 1 commands

# Create some graphs
CreateGraph G1
CreateGraph G2
CreateGraph g
CreateGraph 1

# Create some nodes
CreateNode n1 1
CreateNode n2 2
CreateNode n3 1

# Add some nodes to graph G1
AddNode G1 n1
AddNode G1 n2

# Try to add the node n1 to graph G1
AddNode G1 n1

# Add the node n3 to graph G1
AddNode G1 n3

# Add some nodes to graph 1
AddNode 1 n1
AddNode 1 n2