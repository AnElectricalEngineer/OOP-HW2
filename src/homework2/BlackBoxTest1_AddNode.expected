# Black Box Test 1 commands

# Create some graphs
created graph G1
created graph G2
created graph g
created graph 1

# Create some nodes
created node n1 with cost 1
created node n2 with cost 2
created node n3 with cost 1

# Add some nodes to graph G1
added node n1 to G1
added node n2 to G1

# Try to add the node n1 to graph G1
Node already exists in graph.

# Add the node n3 to graph G1
added node n3 to G1

# Add some nodes to graph 1
added node n1 to 1
added node n2 to 1
